client/constants/index.ts

```diff
@@ -1,12 +1,26 @@
 export * from './keycloak.constants';
 
+export * from '../pages/form-pages/contact-info';
+export * from '../pages/form-pages/general-project-info';
+
+export enum FormTitles {
```

Comment:
Might be best to create a file for these enums and leave the index for exporting